lib/oauth2-loopback.js

```diff
@@ -828,7 +828,7 @@ module.exports = function(app, options) {
     app.post(options.loginPath || '/login',
       passport.authenticate('loopback-oauth2-local',
       {successReturnToOrRedirect: '/',
-        failureRedirect: options.loginPage || '/login'}));
+        failureRedirect: options.loginPage + '?loginFailed' || '/login'}));
```

Comment:
Yeah... you are right! My lack of consideration. So, should we allow options like 'failureMessage' or 'failureFlash' to be passed in, or is there any way to help?